src/index.js

```diff
@@ -184,6 +185,14 @@ class Dropzone extends React.Component {
     this.fileInputEl.click();
   }
 
+  renderChildren() {
+    let { children } = this.props;
```

Comment:
Let's pass `children` to this function and remove `this` completely. In this case this method can be static.